ssg/xml.py

```diff
@@ -445,4 +445,4 @@ def __init__(self, root):
         super(XMLCPEPlatform, self).__init__(root)
 
     def find_all_fact_ref_elements(self):
```

Comment:
```suggestion
    def find_all_check_fact_ref_elements(self):
```
Both `fact-ref` and `check-fact-ref` are a thing, so when you change what the function does, you should also change its name.

Reply:
My intention in this PR is to do only the minimal necessary to fix the issue. When checking this code even for some experiments to include new features, I found many opportunities of improvements. I intend to refactor these files in other PRs. But for this moment, I can also update the function name.

Reply:
Done